src/cloud/live/stage-prod-migrate.md

```diff
@@ -203,16 +203,18 @@ To migrate a database:
 
 1. Create a database dump file in `gzip` format:
 
-   For Starter environments and Pro Integration environments:
-
    ```bash
-   mysqldump -h <database host> --user=<database username> --password=<password> --single-transaction --triggers main | gzip - > /tmp/database.sql.gz
+   mysqldump -h <database host> --user=<database username> --password=<password> --single-transaction --triggers <database name> | gzip - > /tmp/database.sql.gz
    ```
 
-   For Pro Staging and Production environments, the name of the database is in the `MAGENTO_CLOUD_RELATIONSHIPS` variable (typically the same as the application name and username):
+   For Starter environments and Pro Integration environments, use `main` as the name of the database.
+
+   For Pro Staging and Production environments, the name of the database is in the `MAGENTO_CLOUD_RELATIONSHIPS` variable (typically the same as the application name and username).
+
+   If you have configured two-factor authentication it's better to exclude related tables to avoid reconfiguring it after database migration:
```

Comment:
Suggestion for additional clarity.
```suggestion
   If you have configured two-factor authentication on the target environment, it is better to exclude related 2FA tables to avoid reconfiguring it after database migration:
```